sjb/templates/test_case.xml

```diff
@@ -126,12 +126,14 @@ fi
       <defaultExcludes>true</defaultExcludes>
       <caseSensitive>true</caseSensitive>
     </hudson.tasks.ArtifactArchiver>
```

Comment:
@stevekuznetsov Should I also make the `<artifacts>` value exclude the junit artifacts if `junit_analysis` is false?

Reply:
You don't have to